package/src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -222,6 +222,19 @@ const MessageContentWithContext = <
   }
 
   const repliesCurveColor = isMyMessage && !error ? backgroundColor : grey_whisper;
+  const isBorderBottomLeftRadius =
```

Comment:
DRY?

Reply:
To expand a bit on this - it seems like the same thing is happening multiple times/the same code is copied with the only difference being the group style strings. Could you refactor it so we're not repeating ourselves as much?